### PR TITLE
Task 118-119: semeval math challenges 

### PR DESCRIPTION
**Pull request by amirrezamirzaei**
splitting task 104 ([semeval-2019 task10)](https://github.com/allenai/semeval-2019-task-10) into closed and open vocabulary.

### CODE REVIEW

**Review by swarooprm — COMMENTED** [review]
Thanks! Definition of both the tasks look very similar. May be you can add information that are exclusive to the task e.g. in case of task104_semeval_2019_task10_mathematical_answer_generation, you can say something like this "Questions in this task usually contain some variables (e.g. X, Y), equations (e.g. X+Y=3) and some relationships (X is greater than Y). Often you need to find the value of the variable using equations and relationships" and similarly in case of task 118, you can add something that is exclusive to that task.

**Comment by amirrezamirzaei** [comment]
also adding task 119 to this pr because it was made with semeveal-2019 task10 geometric questions. not really sure about because it doesn't contain that many instances (it has 131 instances).

File: tasks/task118_semeval_2019_task10_open_vocabulary_mathematical_answer_generation.json [hunk 1]
```diff
@@ -0,0 +1,1508 @@
...
+    "Contributors": [
+        "Amirreza Mirzaei"
+    ],
+    "Source": [
+        "semeval 2019 task10"
+    ],
+    "Categories": [
+        "Math Question Answering"
+    ],
+    "Definition": "You are given a mathematical question described with an open-ended vocabulary. Questions in this task don't contain mathematical equation but instead involve real-world situation, describing a mathematical problem. you are also given 4 or 5 answer options (associated with \"A\", \"B\", \"C\", \"D\", \"E\"). Things to avoid: 1. Do not generate anything else apart from one of the following characters: 'A', 'B, 'C', 'D', 'E'. Emphasis & Caution: 1. LaTeX mathematical format is used to express equations. 2. Each question is solvable with high school math knowledge. 3. Only give one answer for each question.",
```
Comment (danyaljj) [inline tasks/task118_semeval_2019_task10_open_vocabulary_mathematical_answer_generation.json]: I would simplify it:  "Questions in this task involve ..."

Comment (danyaljj) [reply tasks/task118_semeval_2019_task10_open_vocabulary_mathematical_answer_generation.json]: > LaTeX mathematical format is used to express equations. 

An average human does not know what "LaTeX mathematical format" is. Reformulate/rephrase this.

Comment (danyaljj) [reply tasks/task118_semeval_2019_task10_open_vocabulary_mathematical_answer_generation.json]: `Only give one answer for each question.`  -> 'Give only one answer ...`

File: tasks/task104_semeval_2019_task10_closed_vocabulary_mathematical_answer_generation.json [hunk 2]
```diff
@@ -0,0 +1,4148 @@
...
+    "Contributors": [
+        "Amirreza Mirzaei"
+    ],
+    "Source": [
+        "semeval 2019 task10"
+    ],
+    "Categories": [
+        "Math Question Answering"
+    ],
+    "Definition": "You are given a mathematical question described with a circumscribed mathematical vocabulary. Questions in this task usually contain some variables (e.g. X, Y), equations (e.g. X+Y=3) and some relationships (X is greater than Y). Often you need to find the value of the variable using equations and relationships. you are also given 4 or 5 answer options (associated with \"A\", \"B\", \"C\", \"D\", \"E\"). Things to avoid: 1. Do not generate anything else apart from one of the following characters: 'A', 'B, 'C', 'D', 'E'. Emphasis & Caution: 1. LaTeX mathematical format is used to express equations. 2. Each question is solvable with high school math knowledge. 3. Only give one answer for each question.",
```
Comment (danyaljj) [inline tasks/task104_semeval_2019_task10_closed_vocabulary_mathematical_answer_generation.json]: > you are also 

capitalization 

>  LaTeX mathematical format is used to express equations. 

Same comment as the next task (unclear what "LaTeX mathematical format " means)..

File: tasks/task119_semeval_2019_task10_geometric_mathematical_answer_generation.json [hunk 3]
```diff
@@ -0,0 +1,824 @@
...
+    "Contributors": [
+        "Amirreza Mirzaei"
+    ],
+    "Source": [
+        "semeval 2019 task10"
+    ],
+    "Categories": [
+        "Math Question Answering"
+    ],
+    "Definition": "You are given a geometric mathematical question. questions in this tasks often involve shapes and Geometric Relationships. you are also given 4 or 5 answer options (associated with \"A\", \"B\", \"C\", \"D\", \"E\"). Things to avoid: 1. Do not generate anything else apart from one of the following characters: 'A', 'B, 'C', 'D', 'E'. \\nEmphasis & Caution: 1. LaTeX mathematical format is used to express equations. 2. Each question is solvable with high school geometric knowledge. 3. Only give one answer for each question.",
```
Comment (danyaljj) [inline tasks/task119_semeval_2019_task10_geometric_mathematical_answer_generation.json]: > questions in this tasks 

capitalization 

> 1. LaTeX mathematical format

unclear

**Comment by danyaljj** [comment]
Thanks for the updates. 

In my earlier point regarding "Latex", I  was **not** implying that we should define Latex. 
But rather, we should avoid introducing a new term/concept when it's unnecessary. 
Basically, let's cut to the chase, without saying "LaTeX mathematical format" or "the typesetting software known as LaTeX" which is irrelevant to the task. Just directly talk about equations/symbols/operations. 


Do you think that makes sense?

**Comment by amirrezamirzaei** [comment]
Im not really sure about omitting "Latex" entirely, we need a way to express the equations and other mathematical expressions in this task. if we still use this format to show our equation and just omit "Latex" from the description there might be some confusion about what our equations mean and we cant really express the equations in any other format.

**Comment by danyaljj** [comment]
All that I am saying is that: 
 - many average speakers of English know about +, /, x, etc, since we learn about them at school 
 - Latex is a technical term by a few people 
 - Wherever possible, we should stick to a terminology/framing that is understood by many people.

**Review by swarooprm — COMMENTED** [review]
@amirrezamirzaei Could you address Daniel's comments?

**Comment by danyaljj** [comment]
Upon another look, I see that there are some Latex-specific commands used in the task. For example, `\overline`, `\triangle`, `\frac{}{}`, etc (which is beyond basic algebraic equations). As such, I am fine if we move ahead with this PR as-is.

**Comment by danyaljj** [comment]
Merged with the main branch as well and dropped the repeated instances.

**Pull request merged**